src/main/java/seedu/trackascholar/logic/parser/Prefix.java

```diff
@@ -2,7 +2,7 @@
 
 /**
  * A prefix that marks the beginning of an argument in an arguments string.
- * E.g. 't/' in 'add James t/ friend'.
+ * E.g. 'm/' in 'add James m/ friend'.
```

Comment:
the example of the context does not fit here?

Reply:
Ok thank you it has been resolved.
I changed it to m/' in 'add James m/ Medicine